cetmix_tower_server/security/cx_tower_variable_security.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <!-- user -->
+    <record id="cx_tower_variable_rule_group_user_access" model="ir.rule">
+        <field name="name">Tower Variable: User Access Rule</field>
+        <field name="model_id" ref="model_cx_tower_variable" />
+        <field name="domain_force">[('access_level', '=', '1')]</field>
+        <field name="groups" eval="[(4, ref('cetmix_tower_server.group_user'))]" />
+    </record>
+
+    <!-- manager -->
+    <record id="cx_tower_variable_rule_group_manager_access" model="ir.rule">
+        <field name="name">Tower Variable: Manager Access Rule</field>
+        <field name="model_id" ref="model_cx_tower_variable" />
+        <field name="domain_force">[('access_level', 'in', ['2'])]</field>
```

Comment:
Manager also has access to records with access level less than 2. Check other models rules for example.